inference-engine/src/inference_engine/os/lin/lin_system_conf.cpp

```diff
@@ -81,7 +82,16 @@ int getNumberOfCPUCores() {
             }
         }
     }
-    return CPU_COUNT(&currentCoreSet);
+    int phys_cores = CPU_COUNT(&currentCoreSet);
+    #if (IE_THREAD == IE_THREAD_TBB || IE_THREAD == IE_THREAD_TBB_AUTO)
+    auto core_types = custom::info::core_types();
+    if (bigCoresOnly && core_types.size() > 1) /*Hybrid CPU*/ {
+        const auto little_cores = core_types.front();
+        // assuming the Little cores feature no hyper-threading
+        phys_cores -= custom::info::default_concurrency(custom::task_arena::constraints{}.set_core_type(little_cores));
```

Comment:
The following call returns the default concurrency for big cores only without hyperthreading (with respect to the process affinity)

```suggestion
        phys_cores = custom::info::default_concurrency(custom::task_arena::constraints{}
            .set_core_type(core_types.back())
            .set_max_threads_per_core(1)
        );
```